hosted_control_planes/hcp-destroy/hcp-destroy-ibmpower.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-destroy-ibmz"]
```

Comment:
note that ibmz is referenced, not ibmpower, is that intentional?

Reply:
My bad. Fixed it. Thanks for the review!